joint_limits:
  link_01_name__link_02_name:
    has_position_limits: true
    min_position: -1.0
    max_position: 1.5708
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
  link_02_name__link_03_name:
    has_position_limits: true
    min_position: -1.5708
    max_position: 3.1415
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 5.0
    has_jerk_limits: true
    max_jerk: 100.0
    has_effort_limits: true
    max_effort: 1.0
